Wikipedia/Code/NotificationsCenterCell.swift

```diff
@@ -466,6 +466,19 @@ final class NotificationsCenterCell: UICollectionViewCell {
 
     // MARK: - Public
 
+    fileprivate func setupAccessibility(_ viewModel: NotificationsCenterCellViewModel) {
+        accessibilityAttributedLabel = viewModel.accessibilityText()
+        isAccessibilityElement = true
+        
+        let moreActionAccessibilityActionLabel = viewModel.isRead ? CommonStrings.notificationsCenterMarkAsUnread : CommonStrings.notificationsCenterMarkAsRead
+        let moreAction = UIAccessibilityCustomAction(name: "More", target: self, selector: #selector(tappedMoreAction))
+        let markasReadorUnreadAction = UIAccessibilityCustomAction(name: moreActionAccessibilityActionLabel, target: self, selector: #selector(tappedReadUnreadAction))
+        let isEditing = viewModel.displayState.isEditing
+        
+        accessibilityCustomActions = isEditing ? nil : [moreAction, markasReadorUnreadAction]
```

Comment:
Slightly more performant to not set up those labels if we don't need them. (I think we still need to set as `nil` so we don't get outdated custom actions due to cell reuse, but I'm not 100% sure.)

```suggestion
        if !viewModel.displayState.isEditing {
                let moreActionAccessibilityActionLabel = viewModel.isRead ? CommonStrings.notificationsCenterMarkAsUnread : CommonStrings.notificationsCenterMarkAsRead
                let moreAction = UIAccessibilityCustomAction(name: "More", target: self, selector: #selector(tappedMoreAction))
                let markasReadorUnreadAction = UIAccessibilityCustomAction(name: moreActionAccessibilityActionLabel, target: self, selector: #selector(tappedReadUnreadAction))
        
                accessibilityCustomActions = [moreAction, markasReadorUnreadAction]
        } else {
                accessibilityCustomActions =  nil
        }
```